chall02/gsanchez.py

```diff
@@ -0,0 +1,56 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    gsanchez.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: gsanchez <gsanchez@student.42madrid.com    +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/06 13:11:23 by gsanchez          #+#    #+#              #
+#    Updated: 2020/04/06 13:56:52 by gsanchez         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+import sys
+
+MORSE_CODE = {"a": ".-",
+        "b": "-...",
+        "c": "-.-.",
+        "d": "-..",
+        "e": ".",
+        "f": "..-.",
+        "g": "--.",
+        "h": "....",
+        "i": "..",
+        "j": ".---",
+        "k": "-.-",
+        "l": ".-..",
+        "m": "--",
+        "n": "-.",
+        "o": "---",
+        "p": ".--.",
+        "q": "--.-",
+        "r": ".-.",
+        "s": "...",
+        "t": "-",
+        "u": "..-",
+        "v": "...-",
+        "w": ".--",
+        "x": "-..-",
+        "y": "-.--",
+        "z": "--..",
+		" ": " ",}
+
+if len(sys.argv) != 2:
+	print("usage " + sys.argv[0] + " <a-zA-Z string>")
+	exit()
+if sys.argv[1] == "":
+	print("usage " + sys.argv[0] + " <a-zA-Z string>")
+	exit()
```

Comment:
This is a little redundant don't you think?

Maybe try writing grouped condition, as python's conditions as lazy, the second part of a `or` group won't be evaluated if the first part is true:
```suggestion
if len(sys.argv) != 2 or sys.argv[1] == "":
	print("usage " + sys.argv[0] + " <a-zA-Z string>")
	exit()
```